File: backend/models/activity.py
# backend/models/activity.py
from pydantic import BaseModel
from typing import List


class ActivityResponse(BaseModel):
    """Response model for a single activity"""
    act_keyu: int
    act_code: str
    act_lib: str


class ActivityListResponse(BaseModel):
    """Response model for activity list endpoint"""
    status: str  # "success" or "error"
    message: str
    activities: List[ActivityResponse]

File: backend-go/models/activity.go
package models

// ActivityItem is a single activity entry.
type ActivityItem struct {
	ActKeyu int    `json:"act_keyu"`
	ActCode string `json:"act_code"`
	ActLib  string `json:"act_lib"`
}

// ActivityListResponse is the response body for the activities endpoint.
type ActivityListResponse struct {
	Status     string         `json:"status"` // "success" or "error"
	Message    string         `json:"message"`
	Activities []ActivityItem `json:"activities"`
}
